index.d.ts

```diff
@@ -88,6 +88,8 @@ declare namespace activeWindow {
 	}
 
 	type Result = MacOSResult | LinuxResult | WindowsResult;
+
+	type OpenWindows = Map<string, {processes: Result[]}>;
```

Comment:
```suggestion
	type OpenWindows = Result[];
```

Reply:
Actually. Don't even need the type, just use `Result[]` inline.

Reply:
Map keys are expected to be unique, but process names are not guaranteed to be unique, so this is not going to work.

Reply:
I have make the change for all system, now it will return an array only.